### DISCUSSION TITLE
Alerts App: Add Acknowledge functionality to Group By results

### DISCUSSION THREAD

**Discussion by ccamejo1** [Security-Onion-Solutions/securityonion#9290] [Ideas] [answered]
Would it be possible to add the ability to 'Acknowledge' alerts in 'Group By' results?  Currently, we can escalate from this view, but having the ability to acknowledge alerts would help clear a subsection of the total alerts that may be related to the same IP without having to select them from the events list individually.

![image](https://user-images.githubusercontent.com/13799970/205679469-5e2725f7-bddc-4534-a141-b2d7ad568579.png)

I think it would help teams work through their alerts easier, by dismissing already investigated events by group and leaving unacknowledged events to be investigated.

Keep up the awesome work!

C2

**Comment by dougburks** [comment, marked as answer]
One option might be to enable advanced interface features as shown here:
https://docs.securityonion.net/en/2.3/alerts.html#toggles

With that enabled, the Group Metrics section should retain the Acknowledge button.
